Utilities/Analyzer.cc

```diff
@@ -9,28 +9,20 @@
 #include <iostream>
 #include <string>
 
-#include "DataFormat.cc"
+//#include "DataFormat.cc"
```

Comment:
This include is already threaded through the NanoAODReader, no need to repeat it